src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -55,16 +56,26 @@ object Import {
       Request(None, Right(r), None)
 
     implicit def request2(r: (String, String)): Request =
-      Request(Some(r._1), Left(r._2), None)
+      Try(method(r._1))
+        .map(httpMethod => Request(Some(httpMethod), Left(r._2), None))
+        .getOrElse {
+          r._1 match {
+            case path if path.startsWith("/") =>
+              Request(None, Left(path), Some(r._2))
+            case invalidPath =>
+              throw new IllegalArgumentException(s"The value $invalidPath is not a valid path")
+          }
+        }
```

Comment:
Actually `Try` doesn't have `fold` - should I convert it to `Option` so it can be folded?